Aberrus/Locales/deDE.lua

```diff
@@ -1,106 +1,106 @@
 local L = BigWigs:NewBossLocale("Kazzara, the Hellforged", "deDE")
 if not L then return end
 if L then
-	--L.dread_rift = "Rift" -- Singular Dread Rift
+	L.dread_rift = "Schreckensrisse" -- Singular Dread Rift
 end
 
 L = BigWigs:NewBossLocale("The Amalgamation Chamber", "deDE")
 if L then
-	--L.custom_on_fade_out_bars = "Fade out stage 1 bars"
-	--L.custom_on_fade_out_bars_desc = "Fade out bars which belong to the boss that is out of range in stage 1."
-
-	--L.coalescing_void = "Run Away"
-	--L.shadow_convergence = "Orbs"
-	--L.molten_eruption = "Soaks"
-	--L.swirling_flame = "Tornadoes"
-	--L.gloom_conflagration = "Meteor + Run Away"
-	--L.blistering_twilight = "Bombs + Tornadoes"
-	--L.convergent_eruption = "Soaks + Orbs"
-	--L.shadowflame_burst = "Frontal Cone"
-
-	--L.shadow_and_flame = "Mythic Debuffs"
+	L.custom_on_fade_out_bars = "Verblassender Balken in Phase 1"
+	L.custom_on_fade_out_bars_desc = "Verblassender Balken für entfernten Boss in Phase 1"
+
+	L.coalescing_void = "Weg laufen"
+	L.shadow_convergence = "Orbs"
+	L.molten_eruption = "Soaks"
+	L.swirling_flame = "Tornados"
+	L.gloom_conflagration = "Meteor + Weg laufen"
+	L.blistering_twilight = "Bomben + Tornados
+	L.convergent_eruption = "Soaks + Orbs"
+	L.shadowflame_burst = "Frontal Angriff"
+
+	L.shadow_and_flame = "Mytisch Debuffs"
 end
 
 L = BigWigs:NewBossLocale("The Forgotten Experiments", "deDE")
 if L then
-	--L.rending_charge_single = "1st Charge"
-	--L.massive_slam = "Frontal Cone"
-	--L.unstable_essence_new = "New Bomb"
-	--L.custom_on_unstable_essence_high = "High Stacks Unstable Essence Say Messages"
-	--L.custom_on_unstable_essence_high_desc = "Say messages with the amount of stacks for your Unstable Essence debuff when they are high enough."
-	--L.volatile_spew = "Dodges"
-	--L.volatile_eruption = "Eruption"
-	--L.temporal_anomaly = "Heal Orb"
-	--L.temporal_anomaly_knocked = "Heal Orb Knocked"
+	L.rending_charge_single = "Erste Aufladung"
+	L.massive_slam = "Frontal Angriff"
+	L.unstable_essence_new = "Neue Bombe"
+	L.custom_on_unstable_essence_high = "Zu viele Stacks Instabile Essenz"
+	L.custom_on_unstable_essence_high_desc = "Zu viele Stacks Instabile Essenz erreicht - Dispell"
```

Comment:
We don't mention the word dispel. Please translate the English text.

Reply:
L.custom_on_unstable_essence_high_desc = "Ausgeben einer Nachricht mit der Anzahl der Stapel für den Debuff Instabile Essenz, wenn dieser hoch genug ist."